Source/WebCore/rendering/HitTestResult.cpp

```diff
@@ -964,7 +964,7 @@ HTMLImageElement* HitTestResult::imageElement() const
 bool HitTestResult::isAnimating() const
 {
     if (auto* imageElement = this->imageElement())
-        return imageElement->allowsAnimation();
+        return imageElement->isAnimating();
```

Comment:
Are we sure about this change? Was it a mistake to use `allowsAnimation()` to answer whether the hit tested element `isAnimating()`?